baking/src/tezos_baking/wizard_structure.py

```diff
@@ -304,7 +304,6 @@ def url_is_reachable(url):
 networks = {
     "mainnet": "Main Tezos network",
     "ghostnet": "Long running test network, currently using the mumbai Tezos protocol",
```

Comment:
It seems that you missed this in d20d1aa3589a142f6694e75518348d9971228d98: 
```suggestion
    "ghostnet": "Long running test network, currently using the nairobi Tezos protocol",
```